api/client/options.proto

```diff
@@ -148,6 +150,8 @@ message CommandLineOptions {
     // Remote gRPC source that will deliver to-be-replayed traffic. Each worker will separately
     // connect to this source.
     RequestSource request_source = 26;
+    // Selects and configures a RequestSource plugin.
```

Comment:
This comment doesn't provide any extra information to the user about why and how they would use this field.